src/mixins/amalgamation-mixin.ts

```diff
@@ -344,21 +345,19 @@ export default class AmalgamationMixin extends Vue {
       throw new Error('updatePrepopulatedData(): invalid business')
     }
 
-    // first, fetch new directors/share structure/auth info
-    // NB - business addresses have already been fetched
+    // first, fetch directors and share structure
+    // NB - addresses and auth info have already been fetched
     // NB - make all API calls concurrently without rejection
     // NB - if any call failed, that item will be null
-    const [ directors, shareStructure, authInfo ] =
+    const [ directors, shareStructure ] =
       await Promise.allSettled([
         LegalServices.fetchDirectors(business.identifier),
-        LegalServices.fetchShareStructure(business.identifier),
-        AuthServices.fetchAuthInfo(business.identifier)
```

Comment:
No need to re-fetch auth info as we already have it on the business object (which was last fetched when the draft was restored, or when File and Pay is clicked).